masterlist.yaml

```diff
@@ -28190,3 +28190,49 @@ plugins:
         crc: 0x9E8F3CA5
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 17
+        
+  - name: 'UnforgivingDevices - DD Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'unforgiving Devices DD Patch'
+    after:
+      - 'Devious Devices SE patch.esp'
+      - 'Devious Devices - Assets.esm'
+      - 'Devious Devices - Expansion.esm'
+      - 'Devious Devices - Integration.esm'
+      - 'Devious Devices For Him.esp'
+      - 'Devious Devices - BRRF.esp'
+      - 'Devious Devices - Lore Patch.esp'
```

Comment:
There are a few things to note here about LOOT's sorting. If Plugin A has Plugin B as a master, LOOT will automatically sort Plugin A after Plugin B with no load after rule required. Also, if Plugin C is a master plugin while Plugin D isn't, Plugin D will automatically be loaded after Plugin C as that rule is also hard-coded by the game. If either of these are the case for any of these plugins, please remove them from the list as they are redundant. Otherwise, they can be left, but please sort the remaining plugins alphabetically (ignoring symbols & spaces). All of this applies to the other load after rules in this PR as well, so please double-check them.